src/map_events.c

```diff
@@ -656,7 +656,7 @@ void maintain_my_event_list(struct Dungeon *dungeon)
                 dungeon->event_button_index[i-1] = curr_ev_idx;
                 dungeon->event_button_index[i] = 0;
                 struct Event* event = &game.event[curr_ev_idx];
-                if (!flag_is_set(event->flags,EvF_BtnFirstFall))
+                if (any_flag_is_set(event->flags,EvF_BtnFirstFall))
```

Comment:
I see I accidentally flipped it in #3304 
But why do you change it from flag_is_set to any_flag_is_set? Is simply removing the `!` not enough?

Reply:
It seems that flag_is_set only returns true if the specified flag is the only one set, which isn't what the code was before you flipped it. I tried flag_is_set, but it didn't work. any_flag_is_set matches for what the code was before.

Reply:
> It seems that flag_is_set only returns true if the specified flag is the only one set, which isn't what the code was before you flipped it. I tried flag_is_set, but it didn't work. any_flag_is_set matches for what the code was before.

That concerns me as I have consistently been switching stuff to this function. @eddebaby is this how `flag_is_set` should work, that it checks if that flag, and only that flag, is set?